hibernate-reactive-core/src/main/java/org/hibernate/reactive/provider/service/NoJdbcEnvironmentInitiator.java

```diff
@@ -129,6 +130,9 @@ else if ( url.startsWith("postgresql:") || url.startsWith("postgres:") ) {
 		else if ( url.startsWith("db2:") ) {
 			return  DB297Dialect.class;
 		}
+		else if ( url.startsWith("cockroachdb:") ) {
+			return  CockroachDB192Dialect.class;
```

Comment:
@DavideD I think this should be `CockroachDB201Dialect`, the latest version.

Reply:
Thanks, I will fix it